apps/src/applab/firebaseCounters.js

```diff
@@ -35,7 +37,7 @@ export function updateTableCounters(tableName, rowCountChange, updateNextId) {
       }
       tableData.lastId = (tableData.lastId || 0) + 1;
     }
-    if (tableData.rowCount + rowCountChange > TABLE_ROW_COUNT_LIMIT)  {
+    if (tableData.rowCount + rowCountChange > tableRowCountLimit)  {
```

Comment:
Good catch. Done. Yes, without this there was an implicit ordering assumption that `incrementRateLimitCounters` would always be called first.